src/module_library/AuxBioCro.cpp

```diff
@@ -732,16 +732,41 @@ soilML_str soilML(
  *  into the response of plant production to climate change?: development and
  *  experiments with WIMOVAC: (Windows Intuitive Model of Vegetation response
  *  to Atmosphere & Climate Change)" (University of Essex, 2002)
+ *
+ *  [YH] This function scales the assimilation rate. It should be called the growth
+ *  respiration instead of maintenance respiration, as defined in these papers:
+ *  - Apsim: (https://apsimdev.apsim.info/ApsimX/Documents/AgPastureScience.pdf)
+ *  - Thornley, J. H. M. "Growth, maintenance and respiration: a re-interpretation."
+ *    Annals of Botany 41.6 (1977): 1191-1203.
  */
-double resp(double base_rate, double mrc, double temp)
+double resp(double base_rate, double grc, double temp)
 {
-    double ans = base_rate * (1 - (mrc * pow(2, (temp / 10.0))));
+    double ans = base_rate * (1 - (grc * pow(2, (temp / 10.0))));
```

Comment:
Since we now have function for Q10 temperature response, it would be nice to use it here. There are probably a few other Q10s throughout BioCro. We don't have to standardize them all right now, but this one would be a good start.

Reply:
The Q10 function normalizes `temp` by a `Tref` (usually 25 C). This function does not do that. This was holding me from changing it to a Q10 function.

Reply:
I guess I can make the `Tref` an input to the Q10 function. In this case, it will be 0. This does not need to re-optimize the `grc` parameters.

Reply:
It isn't necessary to reoptimize even if we change `Tref`.  We can calculate what parameter value is equivalent to the original at a different temperature: 

```r
grc_25C = grc_0C * 2^(25/10) 
```

Reply:
Yes. This makes sense. I will however just add the `Tref` as an input for readability.